develop/dev-guide-sample-application-java-hibernate.md

```diff
@@ -1,374 +1,162 @@
+<SimpleTab>
 
-## 第 2 步：获取代码
+<div label="TiDB Serverless">
 
-```shell
-git clone https://github.com/pingcap-inc/tidb-example-java.git
-```
+1. 在 TiDB Cloud Web Console 中，选择你的 TiDB Serverless 集群，进入 **Overview** 页面，点击右上角的 **Connect** 按钮。
 
-与 [Hibernate](https://hibernate.org/orm/) 对比，JDBC 的实现方式并非最优体验。你需要自行编写错误处理逻辑，并且代码无法简单复用。这会使你的代码有些冗余。
+2. 确认窗口中的配置和你的运行环境一致。
 
-此处将以 `6.0.0.Beta2` 版本进行说明。
+    - Endpoint 为 **Public**
+    - Connect With 选择 **General**
+    - Operating System 为你的运行环境。
 
-进入目录 `plain-java-hibernate`：
+    <Tip>如果你在 Windows Subsystem for Linux (WSL) 中运行，请切换为对应的 Linux 发行版。</Tip>
 
-```shell
-cd plain-java-hibernate
-```
+3. 点击 **Generate Password** 生成密码。
 
-目录结构如下所示：
+   <Tip>如果你之前已经生成过密码，可以直接使用原密码，或点击 **Reset Password** 重新生成密码。</Tip>
 
-```
-.
-├── Makefile
-├── plain-java-hibernate.iml
-├── pom.xml
-└── src
-    └── main
-        ├── java
-        │   └── com
-        │       └── pingcap
-        │           └── HibernateExample.java
-        └── resources
-            └── hibernate.cfg.xml
-```
+4. 运行以下命令，将 `env.sh.example` 复制并重命名为 `env.sh`：
 
-其中，`hibernate.cfg.xml` 为 Hibernate 配置文件，定义了：
+    ```bash
+    cp env.sh.example env.sh
+    ```
 
-```xml
-<?xml version='1.0' encoding='utf-8'?>
-<!DOCTYPE hibernate-configuration PUBLIC
-        "-//Hibernate/Hibernate Configuration DTD 3.0//EN"
-        "http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
-<hibernate-configuration>
-    <session-factory>
+5. 复制并粘贴对应连接字符串至 `env.sh` 中。需更改部分示例结果如下。
 
-        <!-- Database connection settings -->
-        <property name="hibernate.connection.driver_class">com.mysql.cj.jdbc.Driver</property>
-        <property name="hibernate.dialect">org.hibernate.dialect.TiDBDialect</property>
-        <property name="hibernate.connection.url">jdbc:mysql://localhost:4000/test</property>
-        <property name="hibernate.connection.username">root</property>
-        <property name="hibernate.connection.password"></property>
-        <property name="hibernate.connection.autocommit">false</property>
+    ```shell
+    export TIDB_HOST='{gateway-region}.aws.tidbcloud.com'
+    export TIDB_PORT='4000'
+    export TIDB_USER='{prefix}.root'
+    export TIDB_PASSWORD='{password}'
+    export TIDB_DB_NAME='test'
+    export USE_SSL='true'
+    ```
 
-        <!-- Required so a table can be created from the 'PlayerDAO' class -->
-        <property name="hibernate.hbm2ddl.auto">create-drop</property>
+    注意替换 `{}` 中的占位符为 **Connect** 窗口中获得的值。
 
-        <!-- Optional: Show SQL output for debugging -->
-        <property name="hibernate.show_sql">true</property>
-        <property name="hibernate.format_sql">true</property>
-    </session-factory>
-</hibernate-configuration>
-```
+    TiDB Serverless 要求使用 secure connection，因此 `USE_SSL` 的值应为 `true`。
 
-`HibernateExample.java` 是 `plain-java-hibernate` 这个示例程序的主体。使用 Hibernate 时，相较于 JDBC，这里仅需写入配置文件地址，Hibernate 屏蔽了创建数据库连接时，不同数据库差异的细节。
+6. 保存文件。
```

Comment:
```suggestion
7. 保存文件。
```

Reply:
<img width="780" alt="image" src="https://github.com/pingcap/docs-cn/assets/1237528/42e47fa8-1b8d-48f2-9a5f-7eedbf7609ce">

It's better to use this format to render an ordered list.